torch/_torch_docs.py

```diff
@@ -2723,9 +2723,9 @@ def merge_dicts(*dicts):
 Also supports batches of matrices, and if :math:`A` is a batch of matrices then the output has the same batch dimensions.
 
 Args:
-    input (Tensor): the input tensor :math:`A` of size :math:`(*, n, n)`,
-                consisting of symmetric positive-definite matrices
-                where :math:`*` is zero or more batch dimensions.
+    input (Tensor): input matrix :math:`u` of size :math:`(*, m, m)`,
```

Comment:
why changing n into m tensor into matrix?
Please use the nomenclature used in the rest of the linalg module, like in [cholesky](https://pytorch.org/docs/stable/generated/torch.linalg.cholesky.html) for example.

Reply:
I used the same description as in [cholesky_solve](https://pytorch.org/docs/stable/generated/torch.cholesky_solve.html?highlight=cholesky#torch.cholesky_solve) which shares exactly the same parameter u

But it is true it could be n instead of m, as  `cholesky` uses n and not m but `cholesky_solve` should probably follow the same nomenclature.

Reply:
The docs for these ops are not particularly clean, as we chose not to port them to `linalg`. The linalg docs are properly curated, so it's best to use them as the reference in style.

If, as a follow up, you want to submit a PR writing the docs for these two ops following the styles of `torch.linalg`, I'd be more than happy to review it :)

Reply:
I added a commit that rewrites these two ops following what is done in `linalg.cholesky`. I let you have a look to the new modifications